Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:12
+RUN npm install code-suggester -g
```

Comment:
we can do this without a Docker container, it might make your iteration a little easier, here are a couple examples:

https://github.com/GoogleCloudPlatform/release-please-action
https://github.com/GoogleCloudPlatform/flaky-service/tree/master/packages/action

If you take the approach of adding an additional package, which simply uses `code-suggester` as a dependency, the lovely thing is the action itself can simply be JavaScript:

https://github.com/GoogleCloudPlatform/release-please-action/blob/main/index.js

Reply:
I'm not sure if JavaScript actions have git natively installed during runtime.That's why I went with a Docker action